test/unit/MerkleDrop.spec.js

```diff
@@ -40,7 +40,7 @@ const getCumulativeAllocation = (addresses, rawAllocations) => {
   );
 };
 
-const mineBlocks = async blockAmount => {
+const mineBlocks = async (blockAmount) => {
```

Comment:
I believe we can remove this function and `forwardBlocks` as well. Both were meant to test the `thresholdBlock` require statement.